pkgs/development/libraries/openbabel/2.nix

```
@@ -25,6 +25,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ zlib libxml2 eigen python3 cairo pcre ];
 
+  cmakeFlags = [ "-DCMAKE_CXX_STANDARD=14" ];
```

```suggestion
  cmakeFlags = lib.optionals stdenv.cc.isClang [
    (lib.cmakeFeature "CMAKE_CXX_STANDARD" "14")
  ];
```
Should we apply this to gcc as well?

---

seems like unnecessary shedding. tested with gcc (built on x64 linux and darwin).